tests/saml/configuration/test_model.py

```diff
@@ -499,22 +499,24 @@ def test_get_settings_returns_correct_result(self):
                 "failOnAuthnContextMismatch": False,
                 "requestedAuthnContextComparison": "exact",
                 "wantNameIdEncrypted": False,
+                "allowRepeatAttributeName": False,
                 "authnRequestsSigned": SERVICE_PROVIDER_WITH_CERTIFICATE.authn_requests_signed
                 or IDENTITY_PROVIDERS[0].want_authn_requests_signed,
                 "logoutResponseSigned": False,
                 "wantMessagesSigned": False,
                 "metadataCacheDuration": None,
+                "rejectDeprecatedAlgorithm": False,
                 "requestedAuthnContext": True,
                 "logoutRequestSigned": False,
                 "wantAttributeStatement": True,
                 "signMetadata": False,
-                "digestAlgorithm": "http://www.w3.org/2000/09/xmldsig#sha1",
+                "digestAlgorithm": "http://www.w3.org/2001/04/xmlenc#sha256",
                 "metadataValidUntil": None,
                 "wantAssertionsSigned": False,
                 "wantNameId": True,
                 "wantAssertionsEncrypted": False,
                 "nameIdEncrypted": False,
-                "signatureAlgorithm": "http://www.w3.org/2000/09/xmldsig#rsa-sha1",
+                "signatureAlgorithm": "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256",
```

Comment:
I think upgrading the `certifi` package updated these fields. I'm not very familiar with the SAML configuration, but hopefully this is ok.

Reply:
Yeah, it should be, SHA1 has been deprecated in favor of SHA256 in general.